File: Assignments/Assignment_27/p1.c
#include<stdio.h>
int factorial(int num){

        int fact=1;
        for(int i=1;i<=num;i++){

                fact*=i;
        }return fact;
}

void main(){

        int num;
        printf("Enter a number:\n");
        scanf("%d",&num);
        printf("Factorial of %d is: %d\n",num,factorial(num));
}

File: JAVA/IF ELSE-1/p1.java
import java.util.*;
class Num{

	public static void main(String[] args){

		System.out.println("Enter a number:");	
		Scanner scan=new Scanner(System.in);
		int x=scan.nextInt();
		if(x>=0){
		
			System.out.println(x+" is positive");
		}else{
		
			System.out.println(x+" is negative");
		}

	}
}
